controllers/applicationvolumereplication_controller.go

```diff
@@ -303,6 +471,18 @@ func (r *ApplicationVolumeReplicationReconciler) extractHomeClusterAndPeerCluste
 	return homeCluster, peerCluster, nil
 }
```

Comment:
(line 459) Is there any case where the subscription is shifting clusters and hence we will have both subStatuses non-empty? Should we protect against that, erroring out till the status has only one entry? More an ACM question at the moment I guess.